modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala

```diff
@@ -34,7 +36,8 @@ final case class ObservationModel(
   constraintSet:        ConstraintSetModel,
   scienceRequirements:  ScienceRequirements,
   plannedTimeSummary:   PlannedTimeSummaryModel,
-  config:               Option[InstrumentConfigModel.Reference]
+  config:               Option[InstrumentConfigModel.Reference],
```

Comment:
Maybe let's keep `config` last?  At some point I may want to figure out a new name.  Maybe `manualConfig`.

Reply:
But no need to change it now, I can move it later.